### ISSUE TITLE
Changing width/height in 2.0.2 causes " attempt to index local 'camera' (a string value)"

### ISSUE THREAD

**Issue by balukin** [veger/TLBE#66]
```
Error while running event TLBE::on_gui_text_changed (ID 2)
TLBE/scripts/camera.lua:279: attempt to index local 'camera' (a string value)
stack traceback:
TLBE/scripts/camera.lua:279: in function 'setWidth'
TLBE/scripts/gui.lua:552: in function <__TLBE__/scripts/gui.lua:515>
```

Changing the width or height of an active camera causes the above error to occur. I tried a quick fix on [my end](https://github.com/veger/TLBE/compare/master...balukin:TLBE:master) and it seems to fix it and allow changes, but I don't have much lua experience and don't know much about parameter matching rules. You may want to take a closer look at this. 

Likely caused by changes in commit d541a9b67455ed9cbf912893e7166811a7a34798

**Comment by veger** [comment]
Thanks, you are 100% right on your deduction. I applied a similar fix as you did (I only created a local variable first, as done in the other functions)

I'll make a new release with this fix.

**Comment by veger** [comment]
It is fixed in v2.0.3
